src/VirtoCommerce.DescriptionExportImportModule.Web/Controllers/Api/ExportController.cs

```diff
@@ -16,11 +17,25 @@ public class ExportController : ControllerBase
     {
         private readonly IUserNameResolver _userNameResolver;
         private readonly IPushNotificationManager _pushNotificationManager;
+        private readonly IProductEditorialReviewSearchService _productEditorialReviewSearchService;
 
-        public ExportController(IPushNotificationManager pushNotificationManager, IUserNameResolver userNameResolver)
+        public ExportController(IPushNotificationManager pushNotificationManager, IUserNameResolver userNameResolver, IProductEditorialReviewSearchService productEditorialReviewSearchService)
         {
             _pushNotificationManager = pushNotificationManager;
             _userNameResolver = userNameResolver;
+            _productEditorialReviewSearchService = productEditorialReviewSearchService;
+        }
+
+        [HttpPost]
+        [Route("count")]
+        public async Task<ActionResult<ExportTotalCountResponse>> GetTotalCount([FromBody] ExportDataRequest request)
+        {
+            var criteria = request.ToSearchCriteria();
+            criteria.Take = 0;
+
+            var searchResult = await _productEditorialReviewSearchService.SearchEditorialReviewsAsync(criteria, deepSearch: true);
+
+            return Ok(new ExportTotalCountResponse { TotalCount = searchResult.TotalCount });
```

Comment:
Maybe use anonymous object? Like 
```suggestion
            return Ok(new { TotalCount = searchResult.TotalCount });
```